langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -119,6 +160,11 @@ public isolated function getName(Element elem) returns string = @java:Method {
 
 # Changes the name of an XML element.
 #
+# ```ballerina
+# xml:Element s = xml `<person>John</person>`;
+# s.setName("student");
```

Comment:
Why isn't `s` being evaluated after?

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18